<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title>Uncovering Earth's Virome</title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="css/notes.css" type="text/css" />
  <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
</head>
<body>
<div id="header">
<h1 class="title">Uncovering Earth's Virome</h1>
<h2 class="author">by Eric J. Ma (MIT)</h2>
</div>
<h1 id="slides-online">Slides Online</h1>
<ul>
<li><a href="http://www.github.com/ericmjl/uncovering-earths-virome">source</a></li>
<li><a href="http://ericmjl.github.io/uncovering-earths-virome/slides.html">slides</a></li>
</ul>
<h1 id="imed-2016">IMED 2016</h1>
<blockquote>
<ul>
<li>21st century disease emergence is accelerating, driven by population growth and related impact on the environment</li>
<li>The world is still ill-prepared to respond to an emerging threat</li>
<li><strong>Our capacity to deploy effective “counter measures” is limited by what we “don’t know” about future threats</strong></li>
<li>Success will require changing from a culture that is “reactive” to one that is “proactive”</li>
<li>We live in an age of Big Data which allows us to think differently about problems and solutions</li>
<li>The Global Virome Project is about Big Data making a Big Impact</li>
</ul>
</blockquote>
<h1 id="motivating-questions">Motivating Questions</h1>
<h2 id="metagenomics-lets-us-answer-these-questions">Metagenomics lets us answer these questions:</h2>
<blockquote>
<ul>
<li>How many viruses are out there?</li>
<li>How can we discover them if we can't grow them?</li>
</ul>
</blockquote>
<h1 id="data-source-and-processing">Data Source and Processing</h1>
<ul>
<li>Publicly available metagenomic data sets from the IMG/M<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a> system.
<ul>
<li>1,729 environmental samples</li>
<li>1,079 host-associated samples</li>
<li>234 engineered samples</li>
</ul></li>
</ul>
<p>Engineered: originating from the &quot;built environment&quot;; check &quot;MicroBENet<a href="#fn2" class="footnoteRef" id="fnref2"><sup>2</sup></a>&quot;.</p>
<h1 id="data-source-and-processing-1">Data Source and Processing</h1>
<h2 id="problem">Problem:</h2>
<ul>
<li>No universal markers, unlike barcode of life.</li>
</ul>
<h2 id="solution">Solution:</h2>
<ul>
<li>Leverage what we know:
<ul>
<li>Protein families from 1,800 manually identified metagenomic viral contigs.</li>
<li>Known viral protein families from isolates.</li>
</ul></li>
<li>Use known viral sequences as bait</li>
</ul>
<h1 id="metagenomic-mining-expands-diversity-of-known-viruses">Metagenomic mining expands diversity of known viruses</h1>
<ul>
<li>Large Viruses
<ul>
<li>596 kb contig, bioreactor</li>
<li>Circular genomes by end overlaps</li>
<li>350-470 kb contig (6 samples)</li>
</ul></li>
</ul>
<h1 id="metagenomic-mining-expands-diversity-of-known-viruses-1">Metagenomic mining expands diversity of known viruses</h1>
<div class="figure">
<img src="./figures/nature19094-f1.jpg" alt="Where new viruses came from." id="fig1" class="class" height="450" />
<p class="caption">Where new viruses came from.</p>
</div>
<h1 id="host-virus-interactions-mapped-by-data-integration">Host-virus interactions mapped by data integration</h1>
<ul>
<li>Viral &quot;Groups&quot;: Project known host-virus interactions.</li>
<li>CRISPR-Cas: identify virus-prokaryote pairings.</li>
<li>Viral tRNA: match to host genomes.</li>
</ul>
<p>This is the one place where I think the paper uncovers new biology.</p>
<h1 id="expanded-view-of-host-virus-interactions">Expanded view of host-virus interactions</h1>
<div class="figure">
<img src="./figures/nature19094-f2.jpg" alt="Viruses mapped to host species." id="fig2" class="class" height="450" />
<p class="caption">Viruses mapped to host species.</p>
</div>
<h1 id="crispr-spacers-targeting-same-virus-found-on-disparate-phyla">CRISPR spacers targeting same virus found on disparate phyla</h1>
<div class="figure">
<img src="./figures/nature19094-f3.jpg" alt="On phage genomes, here&#39;s where CRISPR spacers are found." id="fig3" class="class" height="450" />
<p class="caption">On phage genomes, here's where CRISPR spacers are found.</p>
</div>
<h1 id="viral-distribution-is-clustered-according-to-habitat">Viral distribution is clustered according to habitat</h1>
<div class="figure">
<img src="./figures/nature19094-f4.jpg" alt="Viral distribution patterns in marine and human samples." id="fig4" class="class" height="450" />
<p class="caption">Viral distribution patterns in marine and human samples.</p>
</div>
<h1 id="viruses-can-be-found-basically-all-known-environments">Viruses can be found basically all known environments</h1>
<div class="figure">
<img src="./figures/nature19094-f5.jpg" alt="Habitat distribution of metagenomic viruses." id="fig5" class="class" height="450" />
<p class="caption">Habitat distribution of metagenomic viruses.</p>
</div>
<h1 id="viruses-are-shared-between-geographical-locations">Viruses are shared between geographical locations</h1>
<div class="figure">
<img src="./figures/nature19094-sf10.jpg" alt="Global distribution of viral diversity." id="sfig10" class="class" height="450" />
<p class="caption">Global distribution of viral diversity.</p>
</div>
<h1 id="paper-conclusions-claims">Paper Conclusions &amp; Claims</h1>
<ul>
<li>Metagenomics expands our view of the biosphere.</li>
<li>Comprehensive map of host-virus interactions mined by combining datasets.</li>
</ul>
<h1 id="personal-thoughts">Personal Thoughts</h1>
<h2 id="on-influenza">On Influenza</h2>
<ul>
<li>Influenza's host range is uniquely broad.</li>
</ul>
<h2 id="on-incentives">On Incentives</h2>
<ul>
<li>Incentives for this study and its publication would not be provided by search for therapies.</li>
<li>Yet therapies will very likely come out of this.</li>
</ul>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Integrated Microbial Genomes with Microbial Samples<a href="#fnref1">↩</a></p></li>
<li id="fn2"><p>Microbiology of the Built Environment Network<a href="#fnref2">↩</a></p></li>
</ol>
</div>
</body>
</html>
